app/packs/src/apps/mydb/elements/details/samples/propertiesTab/ElementalComposition.js

```diff
@@ -1,83 +1,56 @@
 import React from 'react';
 
-
 export default class ElementalComposition extends React.Component {
-
-  elementsList(elemental_composition) {
-    let keys = Object.keys(elemental_composition.data);
-    if (keys.length == 0) {
+  elementsList(data) {
+    const keys = Object.keys(data);
+    if (keys.length === 0) {
       return (
         <p>
           Sorry, it was not possible to calculate the elemental
           compositon. Check data please.
         </p>
-      )
+      );
     }
 
-    let elements = [];
-
-    keys.map(function (key, index) {
-      let value = elemental_composition.data[key];
-      elements.push(
-        <span className="data-item" key={key}>
-          <strong>
+    return keys.map((key) => {
+      const value = data[key];
+      return (
+        <span key={key}>
+          <span className="fw-bold">
             {key}
-          </strong>
-          &nbsp;
-          &nbsp;
+          </span>
+          {' '}
```

Comment:
Consider using spacing utility

Reply:
In this case, I think I prefer the actual space. If you copied the text, you'd get the key and value squished together, if we only space them using CSS.